sdk/pom.xml

```diff
@@ -0,0 +1,125 @@
+        </dependency>
+
+        <dependency>
+            <groupId>com.networknt</groupId>
+            <artifactId>json-schema-validator</artifactId>
+            <version>${json.schema.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>${slf4j.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.github.package-url</groupId>
+            <artifactId>packageurl-java</artifactId>
+            <version>${packageurl.version}</version>
+        </dependency>
+
+        <!-- Test dependencies -->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>${junit-jupiter.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>${assertj-core.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>${slf4j.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.moditect</groupId>
+                <artifactId>moditect-maven-plugin</artifactId>
+                <version>1.0.0.Final</version>
+                <executions>
+                    <execution>
+                        <id>add-module-infos</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>add-module-info</goal>
+                        </goals>
+                        <configuration>
+                            <overwriteExistingFiles>true</overwriteExistingFiles>
+                            <failOnWarning>false</failOnWarning>
+                            <module>
+                                <moduleInfo>
+                                    <name>dev.cdevents</name>
+                                    <!-- export everything -->
+                                    <exports>*;</exports>
+                                    <!-- declare services consumed by the artifact -->
+                                    <addServiceUses>true</addServiceUses>
+                                </moduleInfo>
+                            </module>
+                            <jdepsExtraArgs>
+                                <arg>--multi-release=9</arg>
+                            </jdepsExtraArgs>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
I think the library must remain Java 8 compatible. The parent pom defines compiler properties for Java `1.8`. Remove this plugin definition.

Reply:
Removed this plugin

Reply:
Hold on a second. The moditect plugin was added because the codebase was set to Java 8 as a baseline. If the baseline moves to 11 then this plugin may be removed BUT and explicit module descriptor must be added. 

@afrittoli has the Java 8 baseline changed recently?

Reply:
The plugin I requested for removal is the `compiler` plugin because it explicitly overrides source/target compatibility to `11`.

Reply:
I'm not aware of any change from Java 8 to Java 11.

Reply:
I remember we mentioned in the [DEVELOPMENT](https://github.com/cdevents/sdk-java/blob/main/DEVELOPMENT.md) readme "Java 9 is a minimum requirement to build the project",

@aalmiray is that readme need to be updated / any changes needed in the pom.xml ?